src/EPR.Payment.Mopup.Common.Data/Repositories/PaymentsRepository.cs

```diff
@@ -32,14 +32,12 @@ public async Task UpdatePaymentStatusAsync(DataModels.Payment? entity, Cancellat
             await _dataContext.SaveChangesAsync(cancellationToken);
         }
 
-        public async Task<List<DataModels.Payment>> GetPaymentsByStatusAsync(Status status, CancellationToken cancellationToken)
+        public async Task<List<DataModels.OnlinePayment>> GetPaymentsByStatusAsync(Status status, CancellationToken cancellationToken)
         {
             DateTime now = DateTime.UtcNow;
             DateTime updateFrom = now.AddMinutes(-Convert.ToInt32(_configuration["TotalMinutesToUpdate"]));
             DateTime ignoringFrom = now.AddMinutes(-Convert.ToInt32(_configuration["IgnoringMinutesToUpdate"]));
-            var entities = await _dataContext.Payment
-                .Where(a => a.InternalStatusId == Status.InProgress && a.CreatedDate >= updateFrom && a.CreatedDate <= ignoringFrom)
-                .ToListAsync(cancellationToken);
+            var entities = await _dataContext.Payment.Where(a => a.InternalStatusId == Status.InProgress && a.CreatedDate >= updateFrom && a.CreatedDate <= ignoringFrom).ToListAsync();
```

Comment:
cancellationToken is not passed as parameter ToListAsync(). Also retain the formatting into multiple lines

Reply:
Added Token and updated formatting